docs/admin/troubleshooting_connection.rst

```diff
@@ -203,6 +203,39 @@ at the same time. Start ``sd-proxy``, which will bring up ``sd-whonix``.
 
 If this does not resolve the issue, please contact us for assistance.
 
+Customizing Synchronization Timeouts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The SecureDrop Workstation Client application performs a synchronization on
+launch, which may time out in situations where there are a large number of 
+sources on the server. To the extent possible, we recommend reguarly removing
+sources that are no longer actively engaging with you, or any sources that
+appear to be spam.
+
+If you are able to login to the Client application, but sources are not
+appearing, you may be reaching a timeout for the synchronization process.
+To temporarily increase the timeout, and give the system more time to
+finish synchronizing with the server, you can perform the following steps:
+
+1. Log into the Qubes workstation
+2. Start a system Terminal by going to the Qubes Menu, then choose
+   *Terminal Emulator*
+3. Run the following commands::
+   
+       qvm-service --enable sd-app SDEXTENDEDTIMEOUT_600
+       qvm-shutdown sd-app && sleep 5 && qvm-start sd-app
+   
+4. Start the SecureDrop Client application, like normal
```

Comment:
This was the fastest QA procedure in freedomofpress/securedrop-client#1552, but it can also be done without resorting to a terminal, from the "Services" tab of `sd-app`'s "Qube Settings".

Reply:
Thanks for pointing out this potential alternate workflow, @cfm! After looking into this and testing out the Services tab this afternoon, I am still in favor of using the Terminal-based approach, for a few reasons:

* If we decide to merge https://github.com/freedomofpress/securedrop-workstation/pull/857, then `sd-app` (and corresponding Qubes Settings menu) is not directly available to the user
* I found the process of adding the service to be non-intuitive (you have to click the add button, and are just presented with a blank text box, so it's not entirely obvious that that's where "SDEXTENDEDTIMEOUT_600" needs to be added at first)
* Terminal commands are suggested in several other spots within that troubleshooting page in the documentation already (for verifying connectivity and restarting Tor, mainly)

If there's anything I've overlooked, or if others feel differently, I'm certainly not _opposed_ to documenting the GUI-based approach instead, but I do feel the current approach of enabling the service via Terminal is the most straightforward and accessible.

Reply:
Works for me, @nathandyer.  Thanks for looking into it.